### ISSUE TITLE
Don't force import order

### ISSUE THREAD

**Issue by AntoinePrv** [comet-ml/issue-tracking#174]
Hello,

I find forcing `comet_ml` to be imported first to be inconvenient. Especially in unit tests, where comet_ml may be imported as part of a module, but not actually used.
It can also be frustrating to know some things are done automatically but not what those things are ("_Explicit is better than implicit. Simple is better than complex._", The Zen of Python)

Looking at the patch for Pytorch, it seems all it is doing is the automatic loss and automatic model/ graph definition. Both of these things are not required to use comet_ml.

Maybe this could be moved to a sub-module? like `comet_ml.auto_patch`, or even better just a function to run `comet_ml.auto_path()`, or raise a warning rather than an error...

**Comment by Nimrod007** [comment]
Hi, you can use `COMET_DISABLE_AUTO_LOGGING` env var
```
os.environ['COMET_DISABLE_AUTO_LOGGING'] = '1'
```
to disable this option.  in case you want to disable the auto logging feature completely.

its available from version `comet-ml 1.0.39`

**Comment by AntoinePrv** [comment]
Didn't seem to cut it.

**Comment by dsblank** [comment]
@AntoinePrv You'll have to give us a bit more on what you tried, and how it failed. Can you share a small example (say, just the imports) and how you set the variable?

**Comment by Leong1230** [comment]
Hi! I set the COMET_DISABLE_AUTO_LOGGING=1 to disable auto logging, but it still logs a mass of infomation in my log file.
When I use PIL, I met the same logging problem but I can disable it by adding "logging.getLogger('PIL').setLevel(logging.INFO)".
What can I do to disabele the logging infomation from comet, since COMET_DISABLE_AUTO_LOGGING=1 not work for me?
I also tried 'export COMET_DISABLE_AUTO_LOGGING=1' + 'echo $COMET_DISABLE_AUTO_LOGGING'. It didn't work neither.

The importing part of my code is as belowed:

from comet_ml import Experiment
import os
os.environ['COMET_DISABLE_AUTO_LOGGING'] = '1'
import torch
from torch.utils.data import DataLoader
import torchvision
import logging
import argparse
import time
import random
from tensorboardX import SummaryWriter
from torchvision.transforms.transforms import Normalize
